orchestra/git_lfs/__init__.py

```diff
@@ -124,8 +124,12 @@ def find_lfs_files(checkout_dir):
             if sep == b"\0":
                 path, attr, value = next(i), next(i), next(i)
             else:
-                path, attr, value = next(i).rsplit(": ", 2)
-            attr  # shut up pyflakes
+                split_line = next(i).rsplit(b": ", 2)
+                if len(split_line) == 3:
+                    path, attr, value = split_line
+                    attr  # shut up pyflakes
```

Comment:
`path, _, value = split_line`, maybe pyflakes won't complain?

Reply:
I did not write this code so I didn't touch this, actually the linter in my IDE is not happy about this. I'll make this change